src/plugins/intel_cpu/src/transformations/snippets/aarch64/pass/brgemm_to_gemm_cpu.cpp

```diff
@@ -0,0 +1,103 @@
+        return !std::dynamic_pointer_cast<const intel_cpu::tpp::modifier::TensorProcessingPrimitive>(
+            out.get_node_shared_ptr());
+    };
+    auto m_brgemm = ov::pass::pattern::wrap_type<snippets::op::Brgemm>(is_not_tpp);
+
+    auto callback = [=](ov::pass::pattern::Matcher& m) {
+        OV_ITT_SCOPED_TASK(ov::pass::itt::domains::SnippetsTransform, "ov::intel_cpu::pass::BrgemmToGemmCPU")
+        const auto node = m.get_match_root();
+        const auto brgemm = ov::as_type_ptr<snippets::op::Brgemm>(node);
+        const auto brgemm_plugin = ov::as_type_ptr<aarch64::GemmCPU>(node);
+        if (!brgemm || brgemm_plugin) {
+            OPENVINO_THROW("GemmCPU cannot be in body before BrgemmToGemmCPU pass");
+        }
+
+        const auto& brgemm_in0_desc = PortDescriptorUtils::get_port_descriptor_ptr(brgemm->input(0));
+        const auto& brgemm_in1_desc = PortDescriptorUtils::get_port_descriptor_ptr(brgemm->input(1));
+        const auto& brgemm_out_desc = PortDescriptorUtils::get_port_descriptor_ptr(brgemm->output(0));
+
+        const auto dimsMatMulIn0 = snippets::utils::get_planar_pshape(brgemm->input(0));
+        const auto dimsMatMulIn1 = snippets::utils::get_planar_pshape(brgemm->input(1));
+
+        const auto& layout_a = brgemm_in0_desc->get_layout();
+        const auto& layout_b = brgemm_in1_desc->get_layout();
+        const auto& layout_c = brgemm_out_desc->get_layout();
+
+        const auto element_type_a = brgemm->get_input_element_type(0);
+        const auto offset_a = brgemm->get_offset_a();
+        const auto offset_b = brgemm->get_offset_b();
+        const auto offset_c = brgemm->get_offset_c();
+
+        auto gemm_repacking =
+            std::make_shared<aarch64::GemmCopyB>(brgemm->input_value(1), element_type_a, offset_b, 0, layout_b);
+        PortDescriptorUtils::set_port_descriptor(gemm_repacking->input(0), brgemm_in1_desc->get_subtensor(), layout_b);
+        for (const auto& output : gemm_repacking->outputs()) {
+            set_full_port_desc(output);
+        }
+
+        std::shared_ptr<aarch64::GemmCPU> gemm_cpu = std::make_shared<aarch64::GemmCPU>(brgemm->input_value(0),
+                                                                                        gemm_repacking->output(0),
+                                                                                        offset_a,
+                                                                                        0,
+                                                                                        offset_c,
+                                                                                        layout_a,
+                                                                                        std::vector<size_t>{},
+                                                                                        layout_c);
+
+        gemm_cpu->set_friendly_name(brgemm->get_friendly_name());
```

Comment:
please also copy rt info from brgemm to gemm_cpu

Reply:
Good catch! added. Thanks Vladislav!